rundeckapp/grails-app/controllers/rundeck/controllers/UserController.groovy

```diff
@@ -320,10 +318,10 @@ class UserController extends ControllerBase{
         withFormat {
             def xmlClosure = {
                 delegate.'user' {
-                    login(u.login)
-                    firstName(u.firstName)
-                    lastName(u.lastName)
-                    email(u.email)
+                    login(u.getLogin())
+                    firstName(u.getFirstName())
+                    lastName(u.getLastName())
+                    email(u.getEmail())
```

Comment:
I'd prefer we stick to groovy conventions for property getters. `u.login` is the same as `u.getLogin()` but is more readable